### DISCUSSION TITLE
Scam Security Violation

### DISCUSSION THREAD

**Discussion by jazzykjohnson** [community/community#145990] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

The information you gave before is all wrong i cannot report these sites.

People are crying loosing their life because of these links and you want me to wait ??

>URL's hidden by Staff

**Comment by jazzykjohnson** [comment]
https://docs-coinpro-cdn.github.io/en-us/
https://help-coinpro-cdn.gitbook.io/en-us
ttps://docs-coinpro-cdn.gitbook.io/en-us
https://docs-coinpro-cdn.gitbook.io/en-us
https://begin-coinbase.github.io/en-us/
https://docs-coinpro-cdn.gitbook.io/en-us
https://apps--m-start-ledger--auth.webflow.io/
https://docs-coinpro-cdn.github.io/en-us/
https://help-coinpro-cdn.gitbook.io/en-us
https://begin-coinbase.github.io/en-us/
https://docs-trezon----en.gitbook.io/en-us
https://trezorioi-start.mobirisesite.com/
https://auth-moonpay.teachable.com/
https://eng--moonpay--cdn--auth.teachable.com/
https://us-auth.lognitrustcapital.com/
https://auth-sso.lognitrustcapital.com/
https://en-docs-trezors.github.io/en-us/
https://public-trezr.github.io/en-us/
https://doc-trrzr-cdn.gitbook.io/en-us
https://docs-support---ledger-cdn.gitbook.io/en-us
https://home-app---exodu-cdn.gitbook.io/en-us
https://web-exodus-docs.github.io/en-us/

**Comment by Akash1134** [comment]
Hi @jazzykjohnson ,

Thanks for reporting this issue. We'd like to understand what exactly is happening. Please provide the details below:

- Where are these URLs coming from and how have you collected them? How do you know they are the source of phishing?
- What happens when you or your customer click on these URLs? In one of your previous discussions, you mentioned _"During this time, I received a phone call from someone claiming to be from your company. The individual indicated that they noticed my login difficulties and offered to assist me with the issue._" ---> What does that mean? What is an original website account? Could you explain this workflow in detail without posting any URLs?  
- Please do not post these URLs in the community again, as it does not align with our [code of conduct](https://github.com/community/community/blob/c0ea3850fa1bada0be7ba092a2766fefa337ea08/CODE_OF_CONDUCT.md). We've made note of your reports internally.
